src/scripts/components/Loader.js

```diff
@@ -6,7 +6,7 @@ class Loader extends React.Component {
       <div className="loading" ng-hide="currentSurah.ayahs">
         <div className="row">
           <div className="col-md-6 col-md-offset-3 text-center">
-            <img src="//quran-1f14.kxcdn.com/images/loading.gif" />
+            <img src="https://quran-1f14.kxcdn.com/images/loading.gif" />
```

Comment:
With-in the application we should not really care about the protocols. This should be left to web servers to deal with.